sdk/schemaregistry/azure-schemaregistry-avroserializer/dev_requirements.txt

```diff
@@ -0,0 +1,4 @@
+-e ../../identity/azure-identity
```

Comment:
Always put the order of the farthest to the closest:
- devtools
- sdk-tools
- identity
- sr